contrib/check_model.py

```diff
@@ -0,0 +1,131 @@
+import sys
+from collections import defaultdict
+from followthemoney import model
+
+
+IGNORE_DIVERGENT_TYPES = [
+    "author",
+    "organization",
+    "gender",
+    "number",
+    "authority",
+    "duration",
+    "area",
+    "subject",
+    "sender",
+]
+
+IGNORE_DIVERGENT_LABELS = [
+    "parent",
+    "holder",
+    "number",
+    "authority",
+    "criteria",
+    "procedure",
+]
+
```

Comment:
What "divergent types" means: There are two properties with the name "authority" that have different types (haven’t checked it, but probably one is has the `entity` and the other `name` or something like that).

What "divergent labels" means: There are two properties with the same name, but they use different labels in the UI (e.g. `CallForTenders:authority` has the label "Name of contracting authority" while `Sanction:authority` has the label "Authority").

We should be concerned about all of these issues. However, there are some issues that can only be resolved with breaking changes, so I’ve added them to the ignore list for now (otherwise it would break CI).